js/src/forum/utils/UserControls.js

```diff
@@ -124,7 +124,7 @@ export default {
    * @param {string} type
    */
   showDeletionAlert(user, type) {
-    const { username, email } = user.data.attributes;
+    const { displayName: username, email } = user.data.attributes;
```

Comment:
Shouldn't this be `username: displayName`?

Reply:
I tried both and this one works :joy: 

The first name (`displayName`) is the one extracted from `user.data.attributes` and it's then renamed as `username` to keep the remaining of the code like it was before.

I did a few searches to find the correct syntax, this article explains the same https://wesbos.com/destructuring-renaming

I did consider refactoring the code to use a more readable syntax, but then it's two lines that are mostly redundant with each other...

Reply:
Alternate solution, I could change this line to just extract `displayName` and edit line 133 to be `username: displayName`, which might be easier to read?

Reply:
I like that solution. Although you are right: I got this mixed up with graphql aliases.

Reply:
I have refactored those lines. Actually I have no idea why a double spread operator was used on the model internal data instead of retrieving the values from the model in the proper way. I'm now passing the user to the translator directly, so the translator can take care of defining `username`. And I just extract the email separately.